pkg/master/thirdparty_controller_test.go

```diff
@@ -110,15 +136,15 @@ func TestSyncAPIs(t *testing.T) {
 				},
 			},
 			apis: []string{
-				"/apis/example.com",
-				"/apis/example.com/v1",
-				"/apis/example.co",
-				"/apis/example.co/v1",
+				"/apis/example.com/foos",
+				"/apis/example.com/v1/foos",
+				"/apis/example.co/foo",
```

Comment:
I assume you are referring to `apis/example.co` and `apis/example.co/v1`. Yes, these should be added back.